oak_offline_attestation/proto/oak_session_stream_v2.proto

```diff
@@ -0,0 +1,83 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.session.stream.v2;
+
+option java_multiple_files = true;
+option java_package = "oak.session.stream.v2";
+
+message PublicKeyInfo {
+  // The serialized enclave public key.
+  // TODO(#3442): Specify key format.
+  bytes public_key = 1;
+  // The serialized attestation report binding the public key.
+  bytes attestation = 2;
+}
+
+message GetPublicKeyRequest {}
+
+message GetPublicKeyResponse {
+  // The enclave public key.
+  PublicKeyInfo public_key_info = 1;
+}
+
+message InvokeRequest {
+  // Body of the request, encrypted with the enclave public key.
+  // TODO(#3442): Specify encryption format.
+  // TODO(#3442): Also pass client public or symmetric key to use for response encryption. Note that
+  // needs to be at least authenticated by the client, if not encrypted within this field.
```

Comment:
Good point. I think I missed this point in the existing experimental offline attestation implementation.